### DISCUSSION TITLE
Edit knockback

### DISCUSSION THREAD

**Discussion by liexes** [diamond-rip/Eden#206] [Q&A] [answered]
How can I edit the knockback of the combo kit? I don't like your kb.

**Comment by liexes** [comment]
@GoodestEnglish

**Comment by GoodestEnglish** [comment, marked as answer]
You can edit knockback in kit edit menu.
![image](https://user-images.githubusercontent.com/45838505/227048619-0ff45ee0-fd6e-4d83-8f06-2232036d5bfa.png)

**Reply by liexes** [reply]
Fine, but after that? I use papermc and I don't know how to program to make a plugin with its api. I don't understand how to change the rest, yes, I change the name of the kb but after that how do I place the horizontal or vertical?

**Reply by GoodestEnglish** [reply]
As I mentioned in wiki, you cannot edit knockback with papermc. Eden do not handle knockback value. If you do not own the spigot we supported, you can consider using [WindSpigot](https://github.com/Wind-Development/WindSpigot). This spigot allows you to modify knockback like other premium spigot!

**Warning: WindSpigot is archived by their own developer and is no longer supported by them. Eden will not handle any bugs which occurs by WindSpigot.**

**Reply by liexes** [reply]
Any of them is better or similar to papermc? (My server is not modded)

**Reply by GoodestEnglish** [reply]
All of them are either papermc fork, or tacospigot fork. But if you want a free version, windspigot is probably your choice.

**Reply by liexes** [reply]
Ho... Ok Thx
